lib/node_modules/@stdlib/stats/base/dists/triangular/ctor/benchmark/benchmark.js

```diff
@@ -96,16 +107,21 @@ bench( pkg+'::set:a', function benchmark( bm ) {
 	c = 110.0;
 	dist = new Triangular( a, b, c );
 
+	len = bm.iterations;
+	y = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		y[ i ] = ( 100.0*uniform( 0.0, 1.0 ) ) + EPS;
+	}
+
 	bm.tic();
 	for ( i = 0; i < bm.iterations; i++ ) {
-		y = ( 100.0*randu() ) + EPS;
-		dist.a = y;
-		if ( dist.a !== y ) {
+		dist.a = y[ i % len ];
+		if ( dist.a !== y[ i % len ] ) {
 			bm.fail( 'should return set value' );
 		}
 	}
 	bm.toc();
-	if ( isnan( y ) ) {
+	if ( isnan( y[ ( i - 1 ) % len ] ) ) {
```

Comment:
```suggestion
	if ( isnan( y ) ) {
```

This remains unchanged.